app/components/Exchange/MarketPicker.jsx

```diff
@@ -63,7 +63,7 @@ class MarketPickerWrapper extends React.Component {
         let isValidName = !ChainValidation.is_valid_symbol_error(toFind, true);
 
         /* Don't lookup invalid asset names */
-        if (toFind && toFind.length >= 3 && !isValidName) return;
+        if (toFind && toFind.length >= 2 && !isValidName) return;
```

Comment:
This made it impossible to enter inputs longer than 1 character. Fixed in my commit before merging.